phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/common/widgets/lists/icons/IconsListElement.java

```diff
@@ -80,13 +81,14 @@ public IconsListElement(E item, URI miniatureUri, String title, boolean enabled)
             this.setClip(clip);
         });
 
-        final Label label = new Label(title);
-        label.getStyleClass().add("miniatureText");
+        final Region miniature = new Region();
```

Comment:
The class `StaticMiniature` was only a Region containing an image and a css class.
I moved this functionality to the `IconsListElement` class, because it was only 2 lines of code and because this is in lieu to the `CompactListElement` implementation.

The class `StaticMiniature` still exists, because it contains the default miniature uri definitions. I'm not sure where to move them, therefore the I left them where they were and simply moved the class up to the `lists` package